scripts/xcpng-fs-diff.py

```diff
@@ -228,9 +244,14 @@ def compare_data(ref, test, show_diff):
                 continue
 
             if ref_data[dtype][file] != test_data[dtype][file]:
-                print("{} differs: {}".format(dtype, file))
-                if show_diff:
-                    remote_diff(ref_host, test_host, file)
+                print("{} differs: {}".format(dtype, file), end='')
+                if dtype == 'package':
+                    print(" (ref={}, test={})".format(ref_data[dtype][file],
```

Comment:
We tend to prefer f-strings rather than .format (I should have mentioned it in a previous review)

Reply:
OK, let me add a commit to change all `print` in the script